### DISCUSSION TITLE
Warehouse overlay?

### DISCUSSION THREAD

**Discussion by Ouaz** [Keriew/augustus#481] [Ideas]
Maybe it has been already requested, but what do you think about a Warehouse overlay?

It would work the same way than the Food Stocks overlay, but for warehouses:

- Shows only warehouses (obviously)
- Shows only warehousemen on the map (useful to find in a wink where they are, what they are doing, which path they currently take, etc.)

Thanks in advance for the answer!

**Comment by Keriew** [comment]
I think that's a good idea, I added it in 8652292

**Reply by Purakane** [reply]
how do we add these to the game? or do we need to wait for the next update"?

**Reply by Ouaz** [reply]
You can download the latest unstable build for Augustus here: https://github.com/Keriew/augustus#readme

Note: if you are on windows, you need to also download the "assets" folder and replace it in your C3 install directory.

**Reply by crudelios** [reply]
Not just replace the assets, you must delete the old assets folder first before copying over the new one, otherwise you'll have problems.
